server/data-import/get-other-services-csv.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const config = require('config')
+const fs = require('fs')
+const Log = require('log')
+const mongoose = require('mongoose')
+const P = require('bluebird')
+const stringify = require('csv-stringify')
+
+const pregnancyCenterServices = require('../pregnancy-centers/pregnancy-center-services')
+const PregnancyCenterModel = require('../pregnancy-centers/schema/mongoose-schema')
+
+mongoose.Promise = require('bluebird')
+const log = new Log('info')
+
+// TODO: Error handling
+const startDatabase = P.coroutine(function *startDatabase() {
+	yield mongoose.connect(config.mongo.connectionString)
```

Comment:
No changes needed now, but we should be able to use `async/await` here (and remove the comment since we clearly aren't going to add error handling now lol)

Reply:
gotcha, this part I just copied from the server file, but I see what you mean